how-to/graphics/install-nvidia-drivers.md

```diff
@@ -201,6 +201,7 @@ Remove any NVIDIA packages from your system:
 sudo apt --purge remove '*nvidia*${DRIVER_BRANCH}*'
 ```
 
+If you are unsure which `${DRIVER_BRANCH}` to pick for removal you might look at installed nvidia packages and see the different `${DRIVER_BRANCH}` on your system as numbers shared by several packges when you run `apt list --installed | grep nvidia`.
```

Comment:
Maybe it would be useful to tell users that purging the manually installed packaged (not listed as automatic in this list) is enough, because autoremove should take care of the rest? Or is this too much information?

Reply:
I think you are right, but instead of grepping that (language dependent) we should use `apt showmanual` in the first place. Updating this ...